modules/runner/src/main/java/org/apache/ignite/internal/app/IgniteImpl.java

```diff
@@ -1140,8 +1140,7 @@ public class IgniteImpl implements Ignite {
         clientHandlerModule = new ClientHandlerModule(
                 qryEngine,
                 distributedTblMgr,
-                // TODO: IGNITE-20232 The observable timestamp should be different for each client.
-                new IgniteTransactionsImpl(txManager, new HybridTimestampTracker()),
+                txManager,
```

Comment:
can we also fix the last TODO related to IGNITE-20232 (in IgniteTransactionsImpl?):
```
TODO:IGNITE-20232 Remove this method; instead, an interface method should be used.
```
and close INIGTE-20232 when this will be resolved?

Reply:
Yes, I added a comment on the ticket.
After this patch, we won't track an observation timestamp on the server side at all. The only exclusion is JDBC, but about it, I created another ticket IGNITE-24053.